### PR TITLE
add prepare-release scripts

### PR DESCRIPTION
**Pull request by phoracek**
This should make it easier to release new versions of CNAO. In a follow-up PR
we should automate the next step - releasing. So a maintainer would use this
new script to prepare a release PR and automation performs the release once this
PR is merged.

Signed-off-by: Petr Horacek <phoracek@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,8 +255,7 @@ make cluster-down
 
 # Releasing
 
-Steps to create a new release:
-
-1. Test operator, make sure it deploys all components, exposes failures in NetworkAddonsConfig.Status field as well as progressing status of components and "Ready".
-2. Open a new PR with two commits. The first of them adding new released manifests, the second bumping versions in `Makefile`. To make this easier, use `./hack/release.sh <previous version> <released version> <future version> <origin remote> <fork remote>`
-3. Once the PR is merged, tag its **first** commit with proper version name `x.y.z`. This can be done through [GitHub UI](https://github.com/kubevirt/cluster-network-addons-operator/releases/new).
+1. Checkout a public branch
```
Comment (RamLavi) [inline README.md]: where should this branch be created from? is it from master? 
Also, can you be more specific about what you mean by public branch? do you mean create a new branch and pushing to the remote repository in order for it to be public?

Comment (phoracek) [reply README.md]: It is created from the code you are sitting on at the moment. That makes it pretty flexible. I hope to add an automation executed through GitHub as a follow-up.

**Pull request merged**